storage/local/storage.go

```diff
@@ -567,7 +574,13 @@ func (bit *boundedIterator) Close() {
 }
 
 // QueryRange implements Storage.
-func (s *MemorySeriesStorage) QueryRange(_ context.Context, from, through model.Time, matchers ...*metric.LabelMatcher) ([]SeriesIterator, error) {
+func (s *MemorySeriesStorage) QueryRange(ctx context.Context, from, through model.Time, matchers ...*metric.LabelMatcher) ([]SeriesIterator, error) {
+	span, _ := opentracing.StartSpanFromContext(ctx, "QueryRange")
```

Comment:
Indeed - if we want to add any children to this span, it needs to be done with the returned context.  However, I didn't see anything in this function that took a context - the diff shows the original passed in context was ignored.  So when something like `seriesForLabelMatchers` takes a context, we should pass it the returned one.